themes-default/slim/webpack.config.js

```diff
@@ -114,6 +114,9 @@ const webpackConfig = (env, mode) => ({
         runtimeChunk: {
             name: 'vendors'
         },
+        minimizer: [
+            new OptimizeCssAssetsPlugin({})
```

Comment:
This breaks `yarn build`, making all the JS not minimized,
because it overrides the default minimizer option (which I didn't want to touch to begin with).
Just FYI.

Reply:
But before it was added through the plugins section. I think we should override the js minimizer in that case. I can take a look at that.

Reply:
How did you come to the conclusion that you need to override the minimizer?
Also, what webpack 4.x recommendation did you read?

Reply:
After the changes for the copy plugin. It would minimize all static css files. Also in development mode. That's why I moved it to optimize, so it worked as designed

Reply:
Used this. https://medium.com/@davidchristophersally/upgrading-our-build-system-to-webpack4-6b13dd58721a

But honestly, I know jack of webpack

Reply:
Right, I noticed that, it worked okay for development, but not for production.
There's a way around that which I pushed now. Actually minimizing all the CSS files is good for production IMO.

I don't remember finding the default minimizer option (what webpack uses if nothing is provided), and besides that I didn't want to mess with it because it just means more custom stuff that may break with webpack updates or something.

IIRC, webpack 5 will include CSS asset minimizing by default. But who knows when that will come and when/if all the plugins get updated.